app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageActivity.kt

```diff
@@ -41,6 +42,9 @@ import org.kiwix.kiwixmobile.local_file_transfer.LocalFileTransferActivity
 import javax.inject.Inject
 
 class ZimManageActivity : BaseActivity() {
+  override fun injection(coreComponent: CoreComponent) {
+    kiwixActivityComponent.inject(this)
```

Comment:
this should be `cachedComponent.inject(this)`, I needed to store the activityComponent in a field on this object because other objects need access to the component to inject their fields